optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -310,7 +318,7 @@
     """,
     INPUTS_DOCSTRING,
 )
-class OVModelForSeq2SeqLM(OVBaseModelForSeq2SeqLM, GenerationMixin):
+class OVModelForSeq2SeqLM(OVBaseModel, GenerationMixin):
```

Comment:
can we give some description to this class: architecture we expect, format of inputs/outputs and their number?
It will clarify to developers for what cases to utilize this class.

Reply:
Please give some common description in what cases this class is applicable like encoder-decoder arch with text input and output or smth else. So this way I know that I should use this class for some model enabling.
Or please give description that this class presents common/abstract API that child class should implement. And this child class will help to enable some type of models like encoder-decoder arch with text-to-audio.
I am not sure that just having constructor argument description will be helpful to developers.